src/engine/controls/cuecontrol.cpp

```diff
@@ -1633,6 +1657,74 @@ SeekOnLoadMode CueControl::getSeekOnLoadPreference() {
     return static_cast<SeekOnLoadMode>(configValue);
 }
 
+void CueControl::hotcueFocusColorPrev(double v) {
+    if (!v) {
+        return;
+    }
+
+    // Reset CO object
+    m_pHotcueFocusColorPrev->set(0);
```

Comment:
I think this is not required. We will get the update even if one writes 1 again.